lib/pronto/formatter/text_formatter.rb

```diff
@@ -1,44 +1,53 @@
 module Pronto
   module Formatter
-    class TextFormatter
-      include Colorizable
+    class TextFormatter < Base
+      class TextMessageDecorator < SimpleDelegator
```

Comment:
I'm usually not a fan of classes within classes. Could you move outside to `/formatter/text_message_decorator.rb` or give an argument why would you want to keep it inside the `TextFormatter` class?